.travis.yml

```diff
@@ -3,6 +3,10 @@ dist: xenial
 
 language: go
 
+branches:
+  only:
+  - master
```

Comment:
What's the motivation behind having Travis CI enabled only on master?
Wouldn't it cause an approved PR to break master on merging?

Reply:
When the PRs target is master, travis will execute the full build/test, normally.

This ofc can be removed, but maybe we want to target the checking only for the master?

Reply:
Ah I see, my misunderstanding. If @Fulviuus  is fine with this, I'm ok too.